draft-irtf-cfrg-hpke.md

```diff
@@ -355,6 +355,9 @@ HPKE variants rely on the following primitives:
     least `Nsk` bytes of entropy (see {{derive-key-pair}} for discussion)
   - `SerializePublicKey(pkX)`: Produce a byte string of length `Npk` encoding the
     public key `pkX`.
+  - `DeserializePublicKey(enc)`: Parse a byte string of length `Npk` to recover a
+    public key. This function can raise a `DeserializeError` error upon `enc`
+    deserialization failure.
```

Comment:
```suggestion
  - `DeserializePublicKey(pkXm)`: Parse a byte string of length `Npk` to recover a
    public key. This function can raise a `DeserializeError` error upon `pkXm`
    deserialization failure.
```

`enc` is used to denote an encapsulation of a KEM shared secret. Here we're talking about a public key. (An not in all KEMs is `enc` a serialized (ephemeral) public key.) In DHKEM we add `m` to denote a serialized value, so we could do the same here. Although `m` comes from the time when we still used “Marshal” instead of “Serialize”. I don't know if it's still worth to change that to a `s` for example.